ui/nuxeo-page-provider-display-behavior.js

```diff
@@ -447,13 +447,17 @@ export const PageProviderDisplayBehavior = [
     },
 
     _computeLabel() {
-      if (this.loading) {
-        return this.i18n('label.loading');
-      }
-      if (this.filters && this.filters.length > 0) {
-        return this.emptyLabelWhenFiltered ? this.emptyLabelWhenFiltered : this.i18n('label.noResultsWhenFiltered');
-      }
-      return this.emptyLabel ? this.emptyLabel : this.i18n('label.noResults');
+      this._computeLabelDebouncer = Debouncer.debounce(this._computeLabelDebouncer, timeOut.after(500), () => {
```

Comment:
I'm wondering if we should add a comment pointing to the ticket, so we can better understand why we need this. WDYT?

Reply:
🤔 sure, we can add a comment here to explain it. But I wonder if that piece of information wouldn't be redundant since the commit message and blame contain the same information. wdyt?